recipes/protobuf-c/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class ProtobufcConan(ConanFile):
+    name = "protobuf-c"
+    license = "https://github.com/protobuf-c/protobuf-c/blob/master/LICENSE"
+    url = "https://github.com/protobuf-c/protobuf-c"
+    settings = "cppstd", "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+    description = "conan package for protobuf-c"
+    exports_sources = ["patches/**"]
+    _source_subfolder = "source_subfolder"
+    requires = [
+        "protobuf/3.19.2",
+    ]
+    options = {
+        "build_proto3": [True, False],
+        "build_protoc": [True, False],
+        "shared": [True, False],
+        "verbose_makefile": [True, False],
```

Comment:
- add fPIC option please
- try to follow the same order than all other CCI recipes: shared, fPIC, then other options
- why `verbose_makefile` option? We don't care unless I miss something. It's just noise in recipe options.
- I think you can remove `build_` in other options, so proto3 & protoc.